maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/data/Constants.java

```diff
@@ -69,4 +70,27 @@ public class Constants {
             Date.from(Instant.ofEpochSecond(TEST_CONTRIBUTION_CALCULATION_DATETIME.toEpochSecond(ZoneOffset.UTC)));
     public static final SysGenString CONTRIBUTION_BASED_ON = new SysGenString("Means");
 
+    // TODO: Better organise these constants
```

Comment:
To your point about organising, IMHO in both test and prod code we shouldn't have classes with names like `Constants` as they can quickly turn into a dumping ground of loads of unrelated concepts and pieces of data. Furthermore as a result of having so many disparate concerns, it becomes a dependency of many other classes generally increasing the level of coupling in the application.
Another way of looking at it would be, if this class had behaviour in it also we would likely refer to it as a 'God class' ;)

My preference is to define constants with the most restrictive access modifier possible and as narrow scope as possible (to where they are used).
So for these constants I would define them in `TestModelDataBuilder.java` and make them `private`, as they are exclusively used by that class only and concern the creation of test data by the same class.

e.g. [maat Constants.java](https://github.com/ministryofjustice/laa-maat-application/blob/04e586edf62b7638c1baee87faa6c7b3e46805a6/applicationRoot/ccmt-bus/src/main/java/uk/gov/lsc/maat/bus/Constants.java) I don't think could be described as a cohe cohesive class...